.gitignore

```diff
@@ -0,0 +1,6 @@
+vendor/
+./data
+.idea/
+build/
+.env
+data/*
```

Comment:
taková osobní best practice je v repo `data` vůbec nemít a při pouštění testu si tu strukturu vytvořit někde v `/tmp`.

Reply:
Mě přijde fajn to mít na jednom místě. Jinak bych pro různé komponenty měl různé configy a musel bych je furt dokola vytvářet, ne?

Reply:
Rozumím, jakmile jsem si to ošahal, tak konfigurace lokálně už vůbec nepoužívám, občas na debug, ale pro vývoj/testy komponenty setupuju ten data dir uvnitř testů v `/tmp`. A pro ty debugy to mám mimo repozitář, protože stejně většinou jen jeden datadir pro komponentu nestačí :-)

Reply:
Každopádně je tam defaultně nasměrovaný ten `dev` kontejner a tím pádem je nejjednodušší si tam nasypat i tu sandboxovou konfiguraci. Takže nechávám takto prozatím.